option/internaloption/internaloption.go

```diff
@@ -65,3 +65,32 @@ type enableDirectPath bool
 func (e enableDirectPath) Apply(o *internal.DialSettings) {
 	o.EnableDirectPath = bool(e)
 }
+
+// WithDefaultAudience returns a ClientOption that specifies a default audience
+// to be used as the audience field ("aud") for the JWT token authentication.
+//
+// It should only be used internally by generated clients.
+func WithDefaultAudience(audience string) option.ClientOption {
+	return withDefaultAudience(audience)
```

Comment:
FYI the default audience *should* be derived from the Service Name (e.g. `https://SERVICE_NAME/`)

Reply:
It will be derived at client library generation time. This method just facilitates passing the information here.